src/main/java/api/models/request/CrawlingRequest.java

```diff
@@ -0,0 +1,14 @@
+package api.models.request;
+
+public class CrawlingRequest {
```

Comment:
What was this class for, got a bit confused as it wasn't in the class diagram?

Reply:
This one is required by Spring framework, when a request comes the json from the body needs to be parsed to some class. Spring does this automatically by passing it class that corresponds to fields of the json body and that class should have getters and setters and so on. If we just put String native type in the request method, this means that we will have to parse the json manually or use an external lib.